fake-api.js

```diff
@@ -195,6 +195,12 @@ app.get('/stats/aggregate', async (req, res, next) => {
   next();
 });
 
+app.get('/distributionMetrics', async (req, res) => {
```

Comment:
I don't think you need this. We have this line in here later on:

`app.use('/', express.static('test/fake-api/fixtures', { index: 'index.json' }));`

That line will automatically map a request for `distributionMetrics` to a fake API fixture in the `distributionMetrics` folder

Reply:
You are right. 👍

Reply:
8339f0a